### ISSUE TITLE
Projections, napari

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#564]
@sebgoti @maulakhan @grinic 

Currently there is this code in the projections act1 for python:

```python
# %%
# The axes of the max_y_image are 0=z,1=x
# The z axes is squashed thus we enlarge it to make the image isotropic
rescaled_max_y_image = rescale(max_y_image, [dz/dy,1])  
viewer.add_image(rescaled_max_y_image)

# %%
# The max_x_image could be rescaled in the same way
rescaled_max_x_image = rescale(max_x_image, [dz/dy,1])  
viewer.add_image(rescaled_max_x_image)
```

Does this make (always) sense? 

I would think it also is a good option to just add the image with the correct scale to the viewer rather than resampling the data? Note that this is something one cannot do in Fiji and thus one has to rescale, but in napari this is possible and I guess thus it would be cool to use it?

**Comment by grinic** [comment]
Definitely, it would make sense to use the scale argument to show the image with the correct dimensions. I think it’s a good use case of this option, plus ‘Rescale’ is problematic e.g. for large images
